dotnet/src/InternalUtilities/test/HttpMessageHandlerStub.cs

```diff
@@ -8,7 +8,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 
-namespace SemanticKernel.Connectors.UnitTests;
+namespace SemanticKernel.UnitTests;
```

Comment:
```suggestion
```
Proposal; remove namespace completely to not need using in tests classes like in gemini branch.

Reply:
I think is a valid point we may try this in a different PR as the current changes are already adding 20 unrelated files to the pull request.